src/components/file-dropzone/examples/file-dropzone-type-filtering.scss

```diff
@@ -4,16 +4,15 @@
     gap: 1.5rem;
 }
 
-limel-input-field,
-div {
-    min-height: 10rem;
-    background-color: rgb(var(--contrast-100));
-    border-radius: 0.5rem;
+limel-input-field {
+    --textarea-height: 6rem;
```

Comment:
I don't think we should lock the resizing by setting strict height.  It's good if consultants can resize the input field and see that the dropzone also growing.

Reply:
good idea!